pkg/kubelet/images/types.go

```diff
@@ -48,7 +48,7 @@ var (
 // Implementations are expected to be thread safe.
 type ImageManager interface {
 	// EnsureImageExists ensures that image specified by `imgRef` exists.
```

Comment:
This comment needs to be updated because you're renaming `imgRef` to `requestedImage`?